catppuccin-theme.el

```diff
@@ -352,24 +352,33 @@ Must be one of `mocha`, `macchiato`, `frappe`, or `latte`."
                (evil-ex-substitute-replacement :foreground ,ctp-green :underline t)
 
                ;; syntax / font-lock
-               (font-lock-builtin-face :foreground ,ctp-lavender)
-               (font-lock-comment-face :inherit shadow ,@(when catppuccin-italic-comments
-                                                                (list :slant 'italic)))
+               (font-lock-builtin-face :foreground ,ctp-red)
+               (font-lock-comment-face ,@(if catppuccin-italic-comments
+                                             '(:inherit (shadow italic))
+                                           '(:inherit shadow)))
```

Comment:
According to the new style guide, comments, should be overlay0 now, so we should also update shadow to reflect this, and I feel like it fits in this PR well enough, since some of the existing faces are also being updates here.

Reply:
Changed shadow to be overlay0.